src/Platforms/PostgreSQL94Platform.php

```diff
@@ -785,6 +785,12 @@ protected function _getCreateTableSQL($name, array $columns, array $options = []
             }
         }
 
+        if (isset($options['uniqueConstraints']) && ! empty($options['uniqueConstraints'])) {
```

Comment:
The `!empty()` condition is redundant. If it's empty, then the `foreach` will result in no-op which is fine.